src/lib/y2users/linux/users_writer.rb

```diff
@@ -293,9 +319,7 @@ def adapt_home_ownership(user)
       # @param user [User]
       # @param previous_keys [Array<String>] previous auth keys for given user, if any
       # @return [Boolean] true on success
-      def write_auth_keys(user, previous_keys = [])
-        return true unless exist_user_home?(user)
```

Comment:
Why isn't home checked? Would it fail if the user was created without home?

Reply:
I see, an issue is generated when `Yast::Users::SSHAuthorizedKeyring::PathError` is raised.

Reply:
yes and I also think that on system it is not possible to have user without home. at least man page does not mention it and also my own passwd does not have single entry without home. Even one without real home has something like `/bin` or `/sbin`

Reply:
You are right. Checking the code, I see that the home can be created as empty (without skel) but the directory is always created.